### PR TITLE
zh-cn: update the syntax format of `Reflect.set()`

### PR DESCRIPTION
**Pull request by xiaoAn19**
correct spelling mistakes

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/reflect/set/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Reflect/set
 ## 语法
 
 ```plain
-Reflect.set(target, propertyKey, value[, receiver])
+Reflect.set(target, propertyKey, value, receiver)
 ```
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/reflect/set/index.md:14]:
````suggestion
```js-nolint
Reflect.set(target, propertyKey, value)
Reflect.set(target, propertyKey, value, receiver)
```
````

**Review by yin1999 — COMMENTED** [review]
Hey @xiaoAn19. Actually, the original format is correct, the brackets here mean that this parameter is optional.

But it's still worth to update the format, as it's not the same as the [format in the en-US doc](https://github.com/mdn/content/blob/b9c5849612a5beed7f2b8ab5e3c0a336bd585674/files/en-us/web/javascript/reference/global_objects/reflect/set/index.md?plain=1#L16-L19)

File: files/zh-cn/web/javascript/reference/global_objects/reflect/set/index.md (hunk 1 above)
Comment (xiaoAn19) [reply files/zh-cn/web/javascript/reference/global_objects/reflect/set/index.md:14]: Thanks for explaining it to me

**Review by yin1999 — APPROVED** [review]
Thank you @xiaoAn19. Welcome to the community :tada:

**Pull request merged**